docs/getting_started/dask.rst

```diff
@@ -1,91 +1,100 @@
 
-Semantically, Dask sorts the ``index``, which does not allow for user-specified order.
-In Dask's case, this was done for optimization purposes, to speed up other computations
-which involve the row index.
+Non-order preserving
+_____________________
+By default, pandas preserves the order of the DataFrame, so that users can expect a
+consistent, ordered view as they are operating on their DataFrame. However, Dask makes no
+guarantees about the order of rows in the DataFrame. This is because Dask sorts the 
+``index`` for optimization purposes to speed up computations that involve the row index, as 
+a result, it can not support user-specified order.
 
 Modin
 """""
 
-Modin is targeted toward parallelizing the entire pandas API, without exception.
-As the pandas API continues to evolve, so will Modin's pandas API. Modin is intended to
-be used as a drop-in replacement for pandas, such that even if the API is not yet
-parallelized, it still works by falling back to running pandas. One of the key features
+Comprehensive API support
+_________________________
+Modin is targeted toward parallelizing the entire pandas API and currently implements `more than 90% <https://github.com/modin-project/modin#pandas-api-coverage>`_ of the pandas API. Modin acts as a drop-in replacement for pandas, such that even if the API is not yet
+supported, it still works by falling back to running pandas. One of the key features
 of being a drop-in replacement is that not only will it work for existing code, if a
-user wishing to go back to running pandas directly, they may at no cost. There's no
-lock-in: Modin notebooks can be converted to and from pandas as the user prefers.
+user wishes to go back to running pandas directly, they are not locked in to using Modin and can switch between Modin and pandas at no cost. In other words, Modin notebooks can be converted to and from pandas as the user desires.
+
+Eager evaluation
+________________
+Eager evaluation is the default mode of operation for data scientists when working with pandas DataFrames in an interactive environment, such as Jupyter Notebooks. Modin reproduces this familiar behavior by performing all computations as soon as it is issued, so that users can inspect intermediate results and quickly see the results of their computations without having to wait or explicitly trigger computation. This is especially useful during interactive data analysis, where users often iterate on their DataFrame workflows or build up their DataFrame queries in an incremental fashion.
```

Comment:
I guess we should talk about lazy mode of OmniSci here, at least as a sidenote, as otherwise it looks strange when you say "is the default mode" - that implies that other modes exist.

Reply:
Thanks @vnlitvinov! Added description to OmniSci.